teos/src/gatekeeper.rs

```diff
@@ -292,15 +298,15 @@ impl Gatekeeper {
     /// just references to them.
     pub(crate) fn delete_appointments_from_memory(
         &self,
-        appointments: &HashMap<UUID, UserId>,
+        appointments: &HashMap<Locator, UserId>,
```

Comment:
I think this may be significant if the hashmap was big enough, but given the method this may not be the case, would it? It's not like we're going to iterate over milions of items here, but a few thousand at most(?)

Reply:
> I think this may be significant if the hashmap was big enough, but given the method this may not be the case, would it? It's not like we're going to iterate over milions of items here, but a few thousand at most(?)

Not sure about the size of these items tbh. I just thought from the top of my head about what's a good substitute for this pattern.

Reply:
I have gone with 
> Memory & CPU wise, this would better be a Vec<(Locator, UserId)>.

for now.